src/creds/VerifiableCredential.js

```diff
@@ -801,7 +725,7 @@ function transformMetaConstraint(constraintsMeta) {
 
 /**
  * isMatchCredentialMeta
```

Comment:
should probably have an explanation of what this function is doing as it's non-obvious